test/e2e/setup.go

```diff
@@ -207,23 +207,6 @@ func setup(ctx context.Context) error {
 	return nil
 }
 
-func done(ctx context.Context) error {
```

Comment:
Idea: would it be better to do the must-gather logic here, even for local tests? From what I can see, we could call out to the `oc` pkg directly for this (which would save the "download oc" logic, which can fail): https://github.com/openshift/oc/blob/f00e82dc075a867611a999b31285d1866939e2e8/pkg/cli/admin/mustgather/mustgather.go#L280

Reply:
We spoke about this briefly (and I also reached out to a friend who has some experience here) and there isn't really a good way to do the must-gather collection here that we know of, or any kind of must-gather go client at this time.

Last year we worked a little on a must-gather geneva action that borrowed a lot of code from the link you shared. I am not sure if anyone is still working on that, but it might be a good direction to go in the future (with the added benefit for CEE)